packages/helm-charts/oracle/templates/statefulset.yaml

```diff
@@ -121,7 +121,6 @@ spec:
 {{ include "common.env-var" (dict "name" "API_REQUEST_TIMEOUT" "dict" .Values.oracle "value_name" "apiRequestTimeoutMs" "optional" true) | indent 8 }}
 {{ include "common.env-var" (dict "name" "AZURE_HSM_INIT_TRY_COUNT" "dict" .Values.oracle.azureHsm "value_name" "initTryCount") | indent 8 }}
 {{ include "common.env-var" (dict "name" "AZURE_HSM_INIT_MAX_RETRY_BACKOFF_MS" "dict" .Values.oracle.azureHsm "value_name" "initMaxRetryBackoffMs") | indent 8 }}
-{{ include "common.env-var" (dict "name" "BASE_CURRENCY" "dict" .Values.oracle "value_name" "baseCurrency" "optional" true) | indent 8 }}
```

Comment:
@bowd Can you also remove `ALLOW_NOT_CGLD` from here? It was also removed in the clients PR. Also, shall we add `reportTargetOverride` to this file as well?

And if you don't mind the nit let's also move `CURRENCY_PAIR` a bit up to keep the alphabetical order 🙏

Reply:
Very good points - I forgot about reportTargetOverride. Will add 🙏